modules/local/phyloseq.nf

```diff
@@ -8,9 +8,7 @@ process PHYLOSEQ {
         'biocontainers/bioconductor-phyloseq:1.46.0--r43hdfd78af_0' }"
 
     input:
-    tuple val(prefix), path(tax_tsv), path(otu_tsv)
-    path sam_tsv
-    path tree
+    tuple val(prefix), path(tax_tsv), path(otu_tsv), path(sam_tsv), path(tree)
```

Comment:
This is interesting. It's been a point of mine for a while for nf-core modules that it's often better to have a single multi-entry parameter as it allows to keep the arguments synced. But I saw the other day that that's not following the recommendations. Always nice to be proven right! ;-)

Reply:
I actually preferred the previous input line, but I wasnt able to get it running stable without changing it, especially because both, sam_tsv and tree, are optional inputs.